package.json

```diff
@@ -52,9 +52,10 @@
     "@types/ssh2": "^0.5.36",
     "amazon-cognito-identity-js": "^3.0.2",
     "async": "^2.6.1",
-    "aws-api-gateway-client": "^0.2.16",
+    "aws-api-gateway-client": "https://github.com/aregier/aws-api-gateway-client/releases/download/v0.2.16-beta/aws-api-gateway-client-0.2.16.tgz",
```

Comment:
Not sure if it'd be a big deal to an institution running the agent but would it reduce the likelihood of raising their eyebrows to have the gz'd repo coming from the adastradev org here? Obviously it'll be a moot point if we only have to do this for the short term I believe.